CHANGELOG.md

```diff
@@ -15,17 +47,10 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 - Group AND expressions properly to account for nesting [(#18709)](https://github.com/ManageIQ/manageiq/pull/18709)
 - Rename Configuration feature to Main Configuration [(#18707)](https://github.com/ManageIQ/manageiq/pull/18707)
 
-## Unreleased as of Sprint 110 ending 2019-04-29
-
-### Fixed
-- Fix: convert string representation of sizes to numbers when generating SQL for expression [(#18649)](https://github.com/ManageIQ/manageiq/pull/18649)
-
-## Hammer-5
+## Hammer-5 - Released 2019-04-23
 
 ### Added
 - Deprecate invalid custom attribute names [(#18538)](https://github.com/ManageIQ/manageiq/pull/18538)
-- Add ConversionHost validations [(#18277)](https://github.com/ManageIQ/manageiq/pull/18277)
-- Alter ansible_playbook method so that some arguments are optional [(#18323)](https://github.com/ManageIQ/manageiq/pull/18323)
```

Comment:
Why are these moving from the Hammer-5 release to the Hammer-6 one? Shouldn't they stay here?

Reply:
Hmm,...I must have though they were under an unreleased heading. Not sure why they were listed on the spreadsheet as being backported for H-5, and now as H-6. Let me put them back under H-5.

Reply:
@JPrause Because it was backported then reverted in `hammer-5`, then re-backported in `hammer-6`.  You can see the revert in the backport spreadsheet (column A should say revert - for this particular PRs it's row 183/184 "revert PR 18323" and "revert PR 18277), so you need to exclude those from the changelog (unless it's re-backported in the same release).

So PR 18277/18323 are in `hammer-6` only. There are a few other PRs that got reverted in `hammer-5`.

Reply:
Okie doke. Thanks @simaishi